.kokoro-autorelease/common.cfg

```diff
@@ -14,7 +14,22 @@
 
 # Format: //devtools/kokoro/config/proto/build.proto
 
-build_file: "releasetool/.kokoro-autorelease/build.sh"
+# Download trampoline resources.
+gfile_resources: "/bigstore/cloud-devrel-kokoro-resources/trampoline"
+
+build_file: "releasetool/.kokoro/trampoline.sh"
+
+# Configure the docker image for kokoro-trampoline.
+env_vars: {
+    key: "TRAMPOLINE_IMAGE"
+    value: "gcr.io/cloud-devrel-kokoro-resources/node:12-user"
```

Comment:
If possible, can we use `node:14` since that's latest LTS?

Reply:
Done.